lib/analyze_hoc_activity_helper.rb

```diff
@@ -30,12 +30,25 @@ def analyze_day_fast(date)
     " WHERE (finished_at >= '#{day}' AND finished_at < '#{next_day}')" +
     "   OR (pixel_finished_at >= '#{day}' AND pixel_finished_at < '#{next_day}')"
 
+  # Generate a list of Code.org tutorials so that we can generate the count for
+  # Code.org hosted tutorials below.
+  codedotorg_tutorials = []
+  PEGASUS_REPORTING_DB_READONLY.fetch(
+    "SELECT code FROM tutorials WHERE orgname = 'Code.org'"
```

Comment:
I think the orgname is normalized as much as it can be since it can be any alphanumeric string. But I mean this query doesn't account for "code.org" or what if we decided to change it all to "Code Studio" instead of "Code.org" ? Someone changing the gsheet might not know that this is affected, since the gsheet implies it justs what shows on /learn.